server/reload.go

```diff
@@ -30,6 +30,17 @@ func (t *traceOption) Apply(server *Server) {
 	server.Noticef("Reloaded: trace = %v", t.newValue)
 }
 
+// debugOption implements the option interface for the `debug` setting.
+type debugOption struct {
+	newValue bool
+}
+
+// Apply the debug change by reconfiguring the server's logger.
+func (d *debugOption) Apply(server *Server) {
+	server.ConfigureLogger()
```

Comment:
This can work for now but this pattern will have you repeatedly calling this for anything that touches the logger. Eventually will want to call this only once if needed.